code/modules/space_management/turf_reservation.dm

```diff
@@ -0,0 +1,173 @@
+//Yes, they can only be rectangular.
```

Comment:
Lots of commented code in this file, might need a clean-up?

Reply:
A lot of the things left in this file that are commented out will make it easier to have turf reservations for shuttles if/when we make shuttles use turf reservations instead. Still want me to remove them?